.github/workflows/pipeline-deploy-app-test.yaml

```diff
@@ -115,18 +117,29 @@ jobs:
       [
         get-metadata,
         deploy-application-infrastructure,
+        build-and-deploy-application-code,
+        derive-deployment-workspace,
       ]
+    permissions:
+      id-token: write
+      contents: read
+    environment: "test"
     timeout-minutes: 10
     steps:
       - uses: actions/checkout@v4
         with:
           ref: ${{ inputs.git_tag }}
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2.2.0
+        with:
+          role-to-assume: arn:aws:iam::${{ secrets.ACCOUNT_ID }}:role/uec-dos-service-management-github-runner
+          role-session-name: github-pipeline-session
+          aws-region: ${{ vars.AWS_REGION }}
       - name: List Variables
```

Comment:
Can we change the name of this from List Variables to Run Integration Tests so that it is more meaningful when the workflow runs